app/index.js

```diff
@@ -29,9 +29,28 @@ module.exports = yeoman.Base.extend({
       type: String,
     });
 
+    // delete the default options coming from Yeoman
+    delete this._options['help'];
```

Comment:
It will be nice to add a comment explaining why these options need to be deleted 1st